source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -536,6 +540,10 @@ Http::FilterDataStatus JsonTranscoderFilter::decodeData(Buffer::Instance& data,
 
   if (method_->request_type_is_http_body_) {
     request_data_.move(data);
+    if (checkIfDecoderBufferLimitReached(request_data_.length())) {
```

Comment:
I'd be inclined to remove the checkIf in naming here so
if (decoderBufferLimitReached()) and (encoderBufferLimitReached())
as it reads a bit more smoothly.

Reply:
Done